<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QPushButton" name="pClean" >
     <property name="text" >
      <string>Input Clean Configuration </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Select property arrays</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Segment ID array</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="SegmentID" />
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Color array</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="SegmentColor" />
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Select conversion option</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QRadioButton" name="Simplification" >
          <property name="text" >
           <string>Simplification</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="Approximation" >
          <property name="text" >
           <string>Approximation</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QRadioButton" name="Conversion" >
          <property name="text" >
           <string>Conversion</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QRadioButton" name="SimplifyMesh" >
          <property name="text" >
           <string>Simplify Mesh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <layout class="QGridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>Approximation precision                    </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="AngleStep" >
              <property name="decimals" >
               <number>3</number>
              </property>
              <property name="minimum" >
               <double>0.001000000000000</double>
              </property>
              <property name="maximum" >
               <double>90.000000000000000</double>
              </property>
              <property name="value" >
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Simplification Angle Tolerance             </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QDoubleSpinBox" name="AngleTolerance" >
            <property name="layoutDirection" >
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="maximum" >
             <double>45.000000000000000</double>
            </property>
            <property name="value" >
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Activities</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="WithActivity" >
          <property name="text" >
           <string>With Activity</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Activity File      </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="ActivityFile" />
          </item>
         </layout>
        </item>
        <item row="1" column="1" >
         <widget class="QPushButton" name="Browse" >
          <property name="text" >
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Activity step     </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QDoubleSpinBox" name="ActivityStep" >
            <property name="singleStep" >
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0" >
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Date Format</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="dateFormat" >
            <item>
             <property name="text" >
              <string>Select date format</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>YYYY-MM-DD</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>MM/DD/YYYY</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDockWidget" name="wClean" >
     <widget class="QWidget" name="dockWidgetContents_3" >
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" >
          <layout class="QGridLayout" >
           <item row="1" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QCheckBox" name="PieceInvariant" >
               <property name="text" >
                <string>Piece Invariant </string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <layout class="QGridLayout" >
               <item row="0" column="0" >
                <layout class="QGridLayout" >
                 <item row="0" column="0" >
                  <widget class="QLabel" name="label_15" >
                   <property name="text" >
                    <string> Tolerance     </string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="0" column="1" >
                <widget class="QLineEdit" name="Tolerance" />
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="2" column="0" >
            <widget class="QCheckBox" name="ToleranceIsAbsolute" >
             <property name="text" >
              <string>Tolerance is Absolute</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <layout class="QGridLayout" >
             <item row="0" column="0" >
              <widget class="QLabel" name="label_27" >
               <property name="text" >
                <string>Absolute Tolerance</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QLineEdit" name="AbsoluteTolerance" />
             </item>
            </layout>
           </item>
           <item row="0" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="0" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1" >
          <widget class="QCheckBox" name="PointMerging" >
           <property name="text" >
            <string>Point Merging</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QCheckBox" name="ConvertLinesToPoints" >
           <property name="text" >
            <string>Convert Lines To Points</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QCheckBox" name="ConvertPolysToLines" >
           <property name="text" >
            <string>Convert Polys To Lines</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" >
          <widget class="QCheckBox" name="ConvertStripsToPolys" >
           <property name="text" >
            <string>Convert Strips To Polys</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Simplification</sender>
   <signal>toggled(bool)</signal>
   <receiver>AngleTolerance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>111</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>254</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Approximation</sender>
   <signal>toggled(bool)</signal>
   <receiver>AngleStep</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>93</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel" >
     <x>256</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SimplifyMesh</sender>
   <signal>toggled(bool)</signal>
   <receiver>AngleTolerance</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>86</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
